arangodb-net-standard/DocumentApi/DocumentApiClient.cs

```diff
@@ -52,18 +52,20 @@ public DocumentApiClient(IApiClientTransport client, IApiClientSerialization ser
         /// <param name="collectionName"></param>
         /// <param name="document"></param>
         /// <param name="query"></param>
+        /// <param name="serializationOptions"></param>
         /// <returns></returns>
         public virtual async Task<PostDocumentResponse<T>> PostDocumentAsync<T>(
             string collectionName,
             T document,
-            PostDocumentsQuery query = null)
+            PostDocumentsQuery query = null,
+            IApiClientSerializationOptions serializationOptions = null)
```

Comment:
So if this defaults to null, and then passes said null to GetContent, and that doesn't check for null and passes it into the serializer - Would that not cause it to blow up by default?

Reply:
Oh, just saw that it's handled in the Serialize method

Reply:
I left a comment elsewhere suggesting that the API method could be responsible for setting it's own default rather than the serialize.
https://github.com/Actify-Inc/arangodb-net-standard/pull/290#discussion_r501722931